### PR TITLE
docs: update windows setup instructions antivirus hint

### LINKED ISSUE #2158: docs: Change or remove the Defender exclusion in the Windows setup instructions
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

## Description
Due to the fact that some users have [reported issues with anti virus software](https://github.com/JanDeDobbeleer/oh-my-posh/issues/767) the following Microsoft Defender specifics has been added: https://github.com/JanDeDobbeleer/oh-my-posh/commit/89ab82fde06d4400b23054a6f5ecfc6314f2df48

This should be changed.

## Details
Affected files: `docs/docs/installation/windows.mdx`

There are multiple issues with the exclusion that has been added to the installation documention:
1. Using just the file name in the exclusion list is a [common mistake](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/common-exclusion-mistakes-microsoft-defender-antivirus?view=o365-worldwide#using-just-the-file-name-in-the-exclusion-list)
2. [The cmdlet exclude files that have been opened by the process](https://docs.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-process-opened-file-exclusions-microsoft-defender-antivirus#use-powershell-cmdlets-to-exclude-files-that-have-been-opened-by-specified-processes-from-scans)  

    From the [Add-MpPreference docs](https://docs.microsoft.com/en-us/powershell/module/defender/add-mppreference):  
    > Specifies an array of processes, as paths to process images. This cmdlet excludes any files opened by the processes that you specify from scheduled and real-time scanning. **Specifying this parameter excludes files opened by executable programs only. The cmdlet does not exclude the processes themselves.** To exclude a process, specify it by using the ExclusionPath parameter.

## Requested change
I think we should change this to a much more general sentence regarding exclusions, if necessary. (or remove it entirely)

### PR DESCRIPTION
**Pull request by kapsiR**
Resolves #2158

### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes/features)

### Description

This improves the Antivirus hints:
 - Best practise: full path exclusions
 - False positive hint
 - Makes it general, not only Defender specific

### CODE REVIEW

File: docs/docs/installation/windows.mdx [hunk 1]
```diff
@@ -52,11 +52,12 @@ This installs a couple of things:
 
 For the `$PATH` to reload, a restart of your terminal is advised.
 
-:::warning Windows Defender
-To ensure Oh My Posh isn't blocked by Windows Defender, open an elevated PowerShell prompt and run the following command:
+:::warning Antivirus software
+Due to frequent updates of Oh My Posh, Antivirus software occasionally flags it (false positive). To ensure Oh My Posh isn't blocked you can either report it to your favorite Antivirus software as false positive (e.g. [Report a false positive/negative to Microsoft for analysis](https://docs.microsoft.com/en-us/microsoft-365/security/defender/m365d-autoir-report-false-positives-negatives#report-a-false-positivenegative-to-microsoft-for-analysis)) or create an exclusion for it.
```
Comment (JanDeDobbeleer) [inline docs/docs/installation/windows.mdx]: Links need to be at the bottom, and you'll probably beed some newlines to solve the markdownlint remarks 😃

**Review by JanDeDobbeleer — CHANGES_REQUESTED** [review]

File: docs/docs/installation/windows.mdx (hunk 1 above)
Comment (kapsiR) [reply docs/docs/installation/windows.mdx]: Thanks for the fast response! I'll give my best 😉

Comment (kapsiR) [reply docs/docs/installation/windows.mdx]: @JanDeDobbeleer Do you squash it or should I do a force push?

Comment (JanDeDobbeleer) [reply docs/docs/installation/windows.mdx]: Force push is fine!

**Comment by JanDeDobbeleer** [comment]
@All-contributors please add @kapsiR for doc

**Comment by kapsiR** [comment]
@JanDeDobbeleer Thanks for merging and thanks for the ability to work with you, it's a pleasure! (and of course thanks for the great work on Oh My Posh! 💛)

**Pull request merged**